index.html

```diff
@@ -55,22 +55,27 @@ <h1 class="name-intro">
             <div class="works-card1">
                 <div class="child-worksCard-info">
                     <img src="./images/SnapshootPortfolio.png" alt="snapshoot">
-                    <h2 class="prjct-title">TONIC</h2>
-                    <div class="prjct-info">
-                        <h6 class="canopy client-name">CANOPY</h6>
-                        <div class="dot-separator"></div>
-                        <h6 class="client-name">Back End Dev</h6>
-                        <div class="dot-separator"></div>
-                        <h6 class="client-name">2015</h6>
-                    </div>
-                    <p class="description">A daily selection of privately <br> personalized reads; no accounts or <br>
-                        sign-ups required.</p>
-                    <div class="pro-languages">
-                        <p class="html">html</p>
-                        <p class="css">css</p>
-                        <p class="javascript">javascript</p>
+
+                    <div class="card-description">
+                        <h2 class="prjct-title">TONIC</h2>
```

Comment:
- [ ] (OPTIONAL) Kindly apply the correct font color and font family to your cards' headings so as to match the template.
- Your design:
![Annotation 2022-03-24 201921](https://user-images.githubusercontent.com/85113567/159993885-91ed1b7d-6cc5-4ee2-bd7f-28535e222fbb.jpg)
- Figma design:
![Annotation 2022-03-24 200436](https://user-images.githubusercontent.com/85113567/159993921-5c63cf43-2f36-419d-abfa-123c6d67a82a.jpg)